calculator.rb

```diff
@@ -0,0 +1,46 @@
+
+# Taking operation input from user
+
+def user_input
+ arr_op = ["+", "add", "-", "subtract", "*", "multiply", "/", "divide"]
+ puts "Would you like to add, subtract, multiply or divide? "
+ op1 = gets.chomp.to_s
+ while !arr_op.include?(op1)
```

Comment:
I like using `while !arr_op.include?` to control this loop - it feels very clean, and because the potential operations are stored in an array it will be straightforward to add more in the future.